### ISSUE TITLE
repl fails with NSPOSIXErrorDomain error 24

### ISSUE THREAD

**Issue by artemyarulin** [artemyarulin/clojure-clojurescript-buck#25]
If I run `$(buck run repl -- //...)` to process everything in my monorepo eventually planck will stop with error:

```
The operation couldn\U2019t be completed. (NSPOSIXErrorDomain error 24 - Too many open files)
```

**Comment by artemyarulin** [comment]
It turns out it's a planck issue, covered here: https://github.com/mfikes/planck/issues/358

**Comment by artemyarulin** [comment]
Fixed in planck 1.17
